ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml

```diff
@@ -1348,6 +1496,161 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-gcp-op-single-node,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
should this be false as well?

Reply:
Yeah, I think it should. Although I'm not sure if this part of the config is user-adjustable.